CHANGELOG.md

```diff
@@ -23,6 +23,9 @@ to [Semantic Versioning]. Full commit history is available in the
 
 #### Added
 
+- Experimental MuData support for {class}`~scvi.model.MULTIVI` via the method
```

Comment:
Remove Experimental. If we deprecate setup_anndata it is stable and the recommended way.